### ISSUE TITLE
Multiple exporting issues

### ISSUE THREAD

**Issue by schrej** [marktext/marktext#1521]
### Description
We had severe issues with exporting a pretty inline-formula heavy file. We don't exactly know why. Unfortunately the `yarn install` step takes forever for the `cld` building step, so I can't debug it myself.

So I'll just leave the file we were struggling with here: [2.6.md.zip](https://github.com/marktext/marktext/files/3764432/2.6.md.zip)
You can't upload markdown files, that's why I zipped it. We created the entire file using Mark Text.
I'm aware there are a few special characters in there, but if I test them individually, exporting works fine.

- [x/  ] Can you reproduce the issue? Yes and No.

### Steps to reproduce

1. open the attached file
2. try to export to PDF

**Expected behavior:**

I get a PDF file

**Actual behavior:**

Nothing happens.

**Link to an example:**
[2.6.md.zip](https://github.com/marktext/marktext/files/3764432/2.6.md.zip)
(Same as above)

### Versions

- Mark Text: 15.1
- Operating system: macOS 10.14/10.15

**Comment by fxha** [comment]
@schrej Have you install all [prerequisites](https://github.com/marktext/marktext/blob/develop/docs/dev/BUILD.md#prerequisites)? BTW you can launch Mark Text with `--debug` or debug environment variable and click on `View -> Toggle dev tools` to show exceptions.

@Jocs Is this an upstream issue or on our side?

**Stack Trace:**

```
ParseError: KaTeX parse error: Can't use function '$' in math mode at position 2: i$̲. Nein, weil 
Please report this to https://github.com/marktext/marktext/issues.
    at new ParseError (node_modules/katex/dist/katex.js:261:14)
    at Parser.parseFunction (node_modules/katex/dist/katex.js:16506:13)
    at Parser.parseGroup (node_modules/katex/dist/katex.js:16935:21)
    at Parser.parseAtom (node_modules/katex/dist/katex.js:16378:21)
    at Parser.parseExpression (node_modules/katex/dist/katex.js:16243:23)
    at Parser.parse (node_modules/katex/dist/katex.js:16211:22)
    at parseTree (node_modules/katex/dist/katex.js:17195:21)
    at renderToDomTree (node_modules/katex/dist/katex.js:17302:16)
    at Object.renderToString (node_modules/katex/dist/katex.js:17263:16)
    at Object.eval [as mathRenderer] (webpack-internal:///./src/muya/lib/utils/exportHtml.js:41:59)
```

**Comment by Jocs** [comment]
>Is this an upstream issue or on our side?

I haven't looked at this problem in depth, but I'm biased to think it might be wrong parse inline/block math when we export markdown.

**Comment by schrej** [comment]
> @schrej Have you install all [prerequisites](https://github.com/marktext/marktext/blob/develop/docs/dev/BUILD.md#prerequisites)? BTW you can launch Mark Text with `--debug` or debug environment variable and click on `View -> Toggle dev tools` to show exceptions.

Oof, I even looked at the CLI docs but somehow I missed the `--debug` flag, that would've probably saved a few hours. 🙄
I though I had all the dependencies, I might try again another time.

We noticed that for some reason sometimes short "formulas" like `$n$` caused issues, but when you test them individually, they work. During our debugging process we removed lot's of those, they weren't the only thing causing issues, but those issues will probably come up if whatever causes those short ones to fail in this case to show up.
